web/src/locales/en.json

```
@@ -144,5 +147,16 @@
 	"employees.updatePassword.error.incorrectPassword.description": "The password entered does not match your current password.",
 	"employees.error.passwordMismatch": "The password doesn't match the new password",
 	"employees.passwordConstraints": "Password must contain at least 14 characters, including one regular character, one special character, and one number",
-	"confirm": "Confirm"
+	"employees.dateOfBirth.placeholder": "Enter birthdate",
+	"employees.scheduleStart.placeholder": "Enter start time of schedule",
+	"employees.scheduleEnd.placeholder": "Enter end time of schedule",
```

For consistency with PT:
```suggestion
	"employees.scheduleStart.placeholder": "Enter schedule start",
	"employees.scheduleEnd.placeholder": "Enter schedule end",
```